src/vs/workbench/contrib/search/common/searchModel.ts

```diff
@@ -861,12 +861,14 @@ export class FolderMatchWorkspaceRoot extends FolderMatchWithResource {
 		const fileMatchParentParts: URI[] = [];
 		let uri = this.uriParent(rawFileMatch.resource);
 
+		let prev_uri;
```

Comment:
A style thing that we're consistent on is camelCase names instead of underscores, I also recommend always putting a type annotation on variables that aren't immediately assigned.